src/app/private/trainer/directives.tmpl/card.html

```diff
@@ -3,17 +3,26 @@
     class="card__description" 
     ng-class="{true:'card__description--two-lines', false:'card__description--three-lines'}[session.gameModel.comments == null || session.gameModel.comments == '']" >
     <div class="line">{{session.createdTime | date:"dd.MM.yyyy"}}</div>
-    <div class="line line--primary">{{session.name | limitTo: 36}}<span ng-if="session.name.length > 36">...</span></div>
+    <div class="line line--primary">{{session.name | limitTo: 32}}<span ng-if="session.name.length > 32">...</span></div>
```

Comment:
Maybe the standard limit should be stored in a constant...  There is a mix between 36 and 32